RadixWallet/Clients/SensitiveInfoClient/SensitiveInfoClient+Live.swift

```diff
@@ -0,0 +1,19 @@
+extension SensitiveInfoClient: DependencyKey {
+	static var liveValue: SensitiveInfoClient {
+		.init(
+			read: { key in
+				guard let filePath = Bundle.main.path(forResource: "SensitiveInfo", ofType: "plist") else {
+					fatalError("Couldn't find file 'SensitiveInfo.plist'")
```

Comment:
Same here I guess, though it is 99.99% percent guaranteed that the file exists. In general we should not break the app if we cannot configure the AppsFlyer SDK.